third_party/terraform/tests/resource_compute_instance_test.go.erb

```diff
@@ -4750,3 +4797,148 @@ resource "google_compute_resource_policy" "foo" {
 
 `, instance, instance, suffix)
 }
+
+func testAccComputeInstance_subnetworkChange(suffix, instance string) string {
+	return fmt.Sprintf(`
+	data "google_compute_image" "my_image" {
+		family  = "debian-9"
+		project = "debian-cloud"
+	}
+
+	resource "google_compute_network" "inst-test-network" {
```

Comment:
Highly recommend `auto_create_subnetworks = false` on these two- it prevents the network from creating a new subnet per region behind the scenes

Reply:
done